src/scitokens_internal.cpp

```diff
@@ -666,10 +659,12 @@ Validator::get_public_keys_from_web_continue(std::unique_ptr<AsyncStatus> status
         auto now = std::time(NULL);
         // TODO: take expiration time from the cache-control header in the response.
         
-        int next_update_delta, expiry_delta;
-        get_default_expiry_time(next_update_delta, expiry_delta);
-        status->m_next_update = now + next_update_delta;
-        status->m_expires = now + expiry_delta;
+        int next_update_delta = configurer::Configuration::get_next_update_delta();
+        int expiry_delta = configurer::Configuration::get_expiry_delta();
+        next_update = now + next_update_delta;
```

Comment:
next_update and expires doesn't exist.  Go ahead and put them right into the status->m_next_update and status->m_expires variables.